README.md

```diff
@@ -264,6 +264,7 @@ See [CHANGELOG.md](CHANGELOG.md) for detailed list of changes between releases.
 
 See [UPGRADE.md](UPGRADE.md) for information about actions needed when upgrading.
 
+For production use we recommend you to upgrade only when new version is released and not to follow the master branch.
```

Comment:
This is good addition. Can we make it even more prominent, like adding it bolded to installation instructions section? Now it's buried down here and easy to miss.